resources/niri-session

```diff
@@ -1,4 +1,34 @@
-#!/bin/sh
+#!/usr/bin/env sh
+
+print() {
+    case $(basename $SHELL) in
+        fish|bash|zsh)
+            echo -e $@
+            ;;
+        *)
+            # Remove all \x1B escapes from the text.
+            echo $@ | sed 's@\\x1b\[[0-9;]*m@@gi';
+            ;;
+    esac
+}
+
+if ! [ -z $1 ]; then
+    CODE=1
+    if case $1 in --*) true;; *) false;; esac; then
+        if [ "$1" = "--help" ]; then
+            CODE=0
+        else
+            print "\x1b[31;1mError: Unrecognized option $1"
+        fi
+    else
+        print "\x1b[31;1mError: niri-session should be called with no arguments."
```

Comment:
this looks like adding errors for the sake of having errors and doesn't seem to accomplish something